stages/org.osbuild.oscap.remediation

```diff
@@ -128,15 +129,82 @@ def main(tree, options):
     if tailoring is not None:
         cmd.extend(["--tailoring-file", tailoring])
 
+    if arf_results is not None:
+        # run with command in chroot so full path ok
+        cmd.extend(["--results-arf", f"{data_dir}/{arf_results}"])
+
+    cmd.extend(["--results", f"{data_dir}/{xccdf_results}"])
```

Comment:
This result file is now a mandatory artifact as it is a source for fix generation step later on.